Changes

```diff
@@ -2,3 +2,5 @@ Revision history for DBIx-TransactionManager-Distributed
 
 0.01    10/03/2016
         First version, released on an unsuspecting world.
+0.02    10/14/2016
```

Comment:
missed this the first time, but I'd suggest ISO8601 dates here - 2016-10-14 is less ambiguous!